src/ImageProcessor.Web/Configuration/ImageProcessorConfiguration.cs

```diff
@@ -302,40 +302,18 @@ private void LoadImageServices()
                 }
 
                 IImageService imageService = Activator.CreateInstance(type) as IImageService;
-                if (!string.IsNullOrWhiteSpace(config.Prefix))
+                if (imageService != null)
                 {
-                    if (imageService != null)
-                    {
-                        imageService.Prefix = config.Prefix;
-                    }
+                    var name = config.Name ?? imageService.GetType().Name;
```

Comment:
Might not need the GetType().Name thing. Name is required on the config element, so we'll always have one.